forem/forem
# Include Mastodon as social media option with rel=me attribute for verificatiion

## What type of PR is this? (check all applicable)


## Description

Adds Mastodon as an option for social media in `/admin/customization/config` and includes a `rel=me` attribute for verification ([read more here](https://docs.joinmastodon.org/user/profile/#verification))

## Related Tickets & Documents

- Closes #18978

## QA Instructions, Screenshots, Recordings

1. Boot app locally
2. Add a Mastodon URL on `/admin/customization/config` under `Social Media` tab
3. Check the main feed to ensure it's there and the link works properly
4. Add all possible social media options
5. Check the icons don't overflow on the main feed

**Before:**

<img width="742" alt="Screen Shot 2023-01-26 at 11 52 22" src="https://user-images.githubusercontent.com/6045239/214911567-20e9ca40-40db-4a2c-98d5-76a32244b477.png">

**After:**

<img width="597" alt="Screen Shot 2023-01-26 at 11 52 36" src="https://user-images.githubusercontent.com/6045239/214911593-9d2cbccb-f99a-481b-8f41-98c1754ce804.png">

### UI accessibility concerns?

No accessibility concerns, but a small detail to note is that when 6 social media options are enabled the icons overflow. Because of this a small margin is removed. I'm unsure if this is problematic but wanted to make note of it in case others have a suggestion to handle this in a different/better way 🙂 

## Added/updated tests?

- [x] No, and this is why: Existing settings and other specs should cover the existing functionality

## [Forem core team only] How will this change be communicated?


## [optional] Are there any post deployment tasks we need to perform?

Enable the Mastodon Social Media option on DEV

## [optional] What gif best describes this PR or how it makes you feel?

![Elephant sliding down a hill](https://media.giphy.com/media/ccSrsgirRLP0Y/giphy-downsized-large.gif)
